services/object/src/api.js

```diff
@@ -10,4 +11,25 @@ let builder = new APIBuilder({
   context: ['cfg', 'db', 'backends'],
 });
 
+builder.declare({
+  method: 'post',
+  route: '/upload/:name/:projectId',
+  name: 'uploadObject',
+  input: undefined,
+  output: undefined,
+  stability: 'experimental',
+  category: 'Backend',
+  scopes: 'object:upload:<name>/<projectId>',
+  title: 'Upload backend data',
+  description: [
+    'Upload backend data.',
+  ].join('\n'),
+}, async function(req, res) {
+  let { name, projectId } = req.params;
+  const backend = this.backends.forUpload({ name, projectId });
+  let expires = fromNow('1 year');
```

Comment:
The API should probably allow this to be specified.  Perhaps this should be included in the request body?